libs/barista-components/form-field/src/form-field.scss

```diff
@@ -13,11 +13,17 @@
 .dt-form-field-body {
   @include dt-form-control-body();
 
-  .dt-focused & {
+  .dt-focused:not(.dt-form-field-non-box) & {
```

Comment:
I just spotted another & at the end here ... sorry for not pointing that out in the initial review

Reply:
This nesting was not introduced by me, i just added the `:not(.dt-form-field-non-box)`. 
But i can move it top level